Sources/PackageManagerDocs/Documentation.docc/Design/PackageManagerCommunityProposal.md

```diff
@@ -0,0 +1,370 @@
+# Swift Package Manager Community Proposal
```

Comment:
While this is a neat historical document, it doesn't really feel that useful to users in its current form.

Reply:
@bripeticca I'm happy to drop it or keep it - we have https://github.com/swiftlang/swift-package-manager/issues/8600 asking for it to be migrated, and I included it here since it is referenced from the README originally. 

Do we keep or close #8600 as won't fix?

Reply:
Dropped in following update